legal-api/src/legal_api/services/authz.py

```diff
@@ -753,6 +765,13 @@ def has_blocker_future_effective_filing(business: Business, blocker_checks: dict
     return is_fed
 
 
+def has_any_future_effective_filing(business: Business):
+    """Return True if the business has any future effective filing."""
+    now = datetime.utcnow().replace(tzinfo=timezone.utc)
+    pending_filings = Filing.get_filings_by_status(business.id, [Filing.Status.PENDING.value, Filing.Status.PAID.value])
```

Comment:
```suggestion
    now = datetime.now(timezone.utc)
    pending_filings = Filing.get_filings_by_status(business.id, [Filing.Status.PAID.value])
```

Reply:
Before removing PENDING from the query, confirm non-paid filing has to be considered or not

Reply:
Many thanks, Vysakh!